container.go

```diff
@@ -592,6 +612,10 @@ func (c *Container) stop() error {
 		return err
 	}
 
+	if err := c.removeResources(); err != nil {
```

Comment:
By symmetry with `createContainer()`, this call should be placed before `c.detachDevices()`. Indeed, in `createContainer()`, this is called after `c.attachDevices()`

Reply:
agree